### PR TITLE
Feature/service documetation

### PR DESCRIPTION
**Pull request by Andy85de**

### CODE REVIEW

File: docs/For Developer/API/Service.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# Service
+The Service or the API-Service is the interface for you to interact with the UPS. It provides a wide range of CRUD operations to manipulate currently available entities.
+
+## Handling Entity Operations
+The Service utilizes the **Asynchronous Request-Reply Pattern**.The benefit of this pattern is that requests are handled asynchronously. Only operations that manipulate a entity are handled by this pattern.
+
+When an entity is created, deleted, or updated, a **202 (Accepted) HTTP** response code is received, along with a location link in the header. This link allows tracking the progress of the operation. If the entity is not yet ready, a **200 (OK) HTTP** status code is returned. Upon completion of the operation, the location link is redirected using a **302 (Redirect) HTTP** code to the corresponding resource. This approach offers a significant advantage: the service can efficiently handle a high volume of requests simultaneously, while GET requests are processed as usual.
```
Comment (felixkmh) [inline docs/For Developer/API/Service.md:7]: I think currently the status endpoint returns a 202 as long as the operation is pending, not a 200.

File: docs/Installation/Configuration/Sync.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  "Tracing": {
+    "OtlpEndpoint": "",
+    "ServiceName": "userprofile-sync"
+  }
+}
+```
+The service is configured to allow access to all necessary third-party components through the localhost endpoints.
+
+## Configure the UPS-Sync with LDAP Connector
+As of now, we offer support for an LDAP Connector capable of synchronizing data from an existing Active Direcotry for example. The configuration for this feature can be found under the `LDAP` section. We will explane all section step by step. Below is an example configuration for the Active Directory System.
```
Comment (felixkmh) [inline docs/Installation/Configuration/Sync.md:92]:
```suggestion
As of now, we offer support for an LDAP Connector capable of synchronizing data from an existing Active Direcotry for example. The configuration for this feature can be found under the `LDAP` section. We will explain all sections step by step. Below is an example configuration for the Active Directory System.
```

File: docs/Introduction/Features.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Features
+
+* Manages user data including their relations
+* Used entities are hierarchical structured and therefore are stored in a graph.
```
Comment (felixkmh) [inline docs/Introduction/Features.md:4]:
```suggestion
* Used entities are hierarchically structured and therefore are stored in a graph.  
```

File: docs/Introduction/index.md [hunk 4]
```diff
@@ -1,28 +1,36 @@
...
 It consists of three main components:
 
-1. Service: The UPS provides a comprehensive API that allows users to modify entities
+
+1. **Service**: The UPS provides a comprehensive API that allows users to modify entities
 In this context, "modify" refers to performing CRUD (Create, Read, Update, Delete) operations. This API likely includes endpoints for actions such as creating, updating, deleting, and querying user profiles, groups, organizations, roles, and functions.
 
-2. Saga Worker: This component is responsible for validating and creating or modifying entities based on the requests received through the API. It likely handles business logic, data validation, and ensures consistency and integrity of the data. The Saga Worker can be considered the centerpiece of the UPS.
+1. **Saga Worker**: This component is responsible for validating and creating or modifying entities based on the requests received through the API. It likely handles business logic, data validation, and ensures consistency and integrity of the data. The Saga Worker can be considered the centerpiece of the UPS.
```
Comment (felixkmh) [inline docs/Introduction/index.md:13]: "likely" as in "with a high probability" seems strange here.

File: docs/For Developer/API/VolatileData.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+```json
+{
+    "Name": "Sam, Smith",
+    "FirstName": "Sam",
+    "LastName": "Smith",
+    "CreatedAt": "2022-09-11T09:30:11.6796611+02:00",
+    "Id": 235
+}
+```
+Let say we want all user that have the same last name. In a database there can be user with the same last name. The query would look like:
```
Comment (stw914) [inline docs/For Developer/API/VolatileData.md:20]: `Let` => _Let's_ ?

Comment (stw914) [inline docs/For Developer/API/VolatileData.md:20]: > In a database there can be user ...

I don't think the `there` should be _there_ ;)
Suggestion:
> Several user with the same name can be stored in the database.

File: docs/.pages [hunk 6]
```diff
@@ -0,0 +1,7 @@
+nav:
+    - Introduction
+    - Architecture
+    - Installation
+    - Troubleshooting
+    - For Developer
+    - ...
```
Comment (stw914) [inline docs/.pages:7]: Is it possible to skip the `Home` section in the resulting web page?
image

File: docs/For Developer/API/VolatileData.md (hunk 5 above)
Comment (Andy85de) [reply docs/For Developer/API/VolatileData.md:20]: It think you mean user**s**. I  will take the suggestion. It sounds better.

File: docs/.pages (hunk 6 above)
Comment (Andy85de) [reply docs/.pages:7]: I using now the page **Opening Marks** as first page.

File: docs/Installation/Configuration/Service.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
+# Service Configuration
```
[the remark this answers is not in the document]
Comment (Andy85de) [reply docs/Installation/Configuration/Service.md:1]: You are right. The other sections that are not explained will be extended this week.

**Review by stw914 — APPROVED** [review]

**Pull request merged**